avocado/core/jobdata.py

```diff
@@ -170,7 +170,11 @@ def get_resultsdir(logdir, jobid):
     """
     Gets the job results directory using a Job ID.
     """
-    if jobid == 'latest':
+    if os.path.isdir(jobid):
+        return os.path.expanduser(jobid)
+    elif os.path.isfile(jobid):
```

Comment:
Hmm, do we want this? (never noticed we can pass a file and use it's parent as results dir)